src/index.ts

```diff
@@ -3,7 +3,7 @@ import * as bodyParser from 'body-parser';
 import * as compression from 'compression';
 import * as cookieParser from 'cookie-parser';
 import cookieSession = require('cookie-session');
-import * as _express from 'express';
+import * as express from 'express';
```

Comment:
Typescript was always clever enough to elide unused types (or certainly for a very long time), this was also about making sure they weren't used for anything other than types unintionally)

Reply:
You're right, I'll fix the commit message and PR description.